pkg/apis/internal.acorn.io/v1/event.go

```diff
@@ -40,7 +40,7 @@ type EventInstance struct {
 	Description string `json:"description,omitempty"`
 
 	// Observed represents the time the Event was first observed.
-	Observed metav1.Time `json:"observed"`
+	Observed metav1.MicroTime `json:"observed" wrangler:"type=string"`
```

Comment:
Just curious, why did you need to add this `wrangler` tag to this struct field?

Reply:
We use `wrangler` to generate the internal CRDs at runtime, and without this, it generates an incorrect openapi spec for `MicroTime` that breaks serialization and conversion; general chaos.